main/guides/getting-started/README.md

```diff
@@ -6,13 +6,11 @@ Welcome! We'll start with a basic _dapp_; that is: a JavaScript contract and ass
 style="border: 1px solid" width="300"
 src="https://github.com/Agoric/documentation/assets/150986/36a87384-0148-4f9e-8606-e176bd7880b3" />
 
-```js
-/** @param {ZCF<{joinPrice: Amount}>} zcf */
-export const start = async zcf => {
-  const { joinPrice } = zcf.getTerms();
-...
-}
-```
+::: details gameAssetContract.js listing
+
+<<< @/snippets/zoe/src/gameAssetContract.js#file
+
+:::
 
 As we begin, note that help is not far away:
 
```

Comment:
_note_

As mentioned before

> We support any long-term support (LTS) versions of Node.js.

on line 27 is no longer valid given incompatibility issues with node 20(https://github.com/Agoric/agoric-sdk/issues/8636)

PS: commenting on this link because GitHub wouldn't allow me to comment on line 27 directly

Reply:
Well, the policy is still to support Node LTS. It's just that we've got a bug. The bug should probably be noted, along with a work-around.

Note that this part of this PR is sort of obsolete in favor of #894 .